tests/e2e/util.go

```diff
@@ -6823,3 +6827,92 @@ func getVmdkPathFromVolumeHandle(sshClientConfig *ssh.ClientConfig, masterIp str
 	vmdkPath := result.Stdout
 	return vmdkPath
 }
+
+// checkVcServicesHealthPostReboot returns waitErr, if VC services are not running in given timeout
+func checkVcServicesHealthPostReboot(ctx context.Context, host string, timeout ...time.Duration) error {
```

Comment:
Also, there are multiple files where we have called vc rebbot, you have to call this util in all such places

Reply:
added it in afterEach block with isVcRebooted flag.